src/game.h

```diff
@@ -462,8 +461,6 @@ class game
 
         item_location inv_map_splice( item_filter filter, const std::string &title, int radius = 0,
```

Comment:
I'm going to get rid of both. `inv_map_splice()` that accepts `item_location_filter` simply hasn't been used after these changes so I removed it.